src/components/cylc/analysis/TimeSeries.vue

```diff
@@ -0,0 +1,459 @@
+<!--
+Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+  <Teleport
+    v-if="sortInputTeleportTarget"
+    :to="sortInputTeleportTarget"
+  >
+    <div class="d-flex flex-grow-1 col-gap-1">
+      <v-autocomplete
+        multiple
+        chips
+        closable-chips
+        clearable
+        placeholder="Search"
+        :items="tasks"
+        v-model="displayedTasks"
+        label="Select tasks"
+        ref="selectTasks"
+        @update:search="updateSelectionOptions"
+      >
+        <template
+          v-slot:prepend-item
+        >
+          <v-list-item
+            @click="selectSearchResults"
+          >
+            Select all search results
+          </v-list-item>
+          <v-list-item
+            @click="deselectSearchResults"
+          >
+            Remove all search results
+          </v-list-item>
+          <v-divider class="mt-2"></v-divider>
```

Comment:
A suggestion to make this more compact

![image](https://github.com/cylc/cylc-ui/assets/61982285/e7c4dc65-4504-40ee-b827-923559d663a7)


```suggestion
        <template v-slot:prepend-item>
          <v-card-actions class="mt-n2">
            <v-btn @click="selectSearchResults">
              Select all
            </v-btn>
            <v-btn @click="deselectSearchResults">
              Deselect all
            </v-btn>
          </v-card-actions>
          <v-divider/>
```